examples/connectionWithCancellation.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * run with:
+
+ *   `WHEROBOTS_API_KEY=<api key> node -r @swc-node/register examples/connectionWithCancellation.ts`
+ * 
+ * or for verbose logging:
+ * 
+ *   `NODE_DEBUG="wherobots-sql-driver" WHEROBOTS_API_KEY=<api key> node -r @swc-node/register examples/connectionWithCancellation.ts`
+ */
+
+import { Connection, Runtime } from "@/index";
+import { Utf8 } from "apache-arrow";
+
+(async () => {
+  const conn = await Connection.connect({
+    runtime: Runtime.SEDONA,
```

Comment:
Can we support the new runtime names?

Reply:
good call, will fix that in a follow-on pr